server/superdesk/tests.py

```diff
@@ -146,12 +146,16 @@ def get_prefixed_url(current_app, endpoint):
 
 
 def setup_db_user(context, user):
+    """
+    Setup the user for the DB authentication.
+    :param context: test context
+    :param dict user: user
+    """
     user = user or test_user
     with context.app.test_request_context(context.app.config['URL_PREFIX']):
         original_password = user['password']
 
-        if user.get('user_type') is None:
-            user['user_type'] = 'administrator'
+        user.setdefault('user_type', 'administrator')
```

Comment:
`user_type` is not provided. There are some test where `user_type` is provided via the step definition. It should be fine. merging the code for now as it is blocker for our ldap configuration.